src/Modules/Workspaces/Commands.Workspaces/help/Add-PowerBIWorkspaceUser.md

```diff
@@ -0,0 +1,106 @@
+---
+external help file: Microsoft.PowerBI.Commands.Workspaces.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Workspaces
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIWorkspaceUser
+
+## SYNOPSIS
+Gives permissions to a given user to a Power BI workspace 
+
+## SYNTAX
+
+```
+Add-PowerBIWorkspaceUser [-Scope <PowerBIUserScope>] -Id <Guid> -UserEmailAddress <String>
+ -UserAccessRight <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Add-PowerBIWorkspaceUser -Scope Organization -GroupId 23FCBDBD-A979-45D8-B1C8-6D21E0F4BE50 -UserEmailAddress john@contoso.com
+ -UserAccessRight Admin
+```
+
+## PARAMETERS
+
+### -Id
+Group or Workspace Id for which user has to be added
+
+```yaml
+Type: Guid
+Parameter Sets: (All)
+Aliases: GroupId, WorkspaceId
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Scope
+Individual or Organization
+
+```yaml
+Type: PowerBIUserScope
+Parameter Sets: (All)
+Aliases:
+Accepted values: Individual, Organization
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UserAccessRight
+Permissions to assign to the user
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UserEmailAddress
+User Principal name of the user who is to be given permissions to the Group/Workspace
```

Comment:
Is it UPN or e-mail here?

Reply:
As discussed updated the parameter name to UserPrincipalName, added alias for UserEmailAddress and updated documentation.